cli.js

```diff
@@ -1549,10 +1568,68 @@ async function execute(network, action, ...params) {
     try {
       const liquidationCallTx = await lendingPool.methods
         .liquidationCall(collateralAssetAddr, debtAssetAddr, riskUser, debtToCover, receiveAToken)
-        .send({ from: user, gas: 2000000 });
+        .send({ from: user, gas: DEFAULT_GAS });
       console.log('liquidationCall: ', liquidationCallTx.transactionHash);
     } catch (err) {
-      console.log(`Cannot liquidate user ${riskUser}: `, err.message);
+      console.error(`Cannot liquidate user ${riskUser}: `, err.message);
+    }
+
+    return;
+  }
+
+  if (action === 'repaydelegation') {
+    const delegator = params[0];
+    const asset = tokens[params[1].toLowerCase()];
+    const assetAddr = asset.options.address;
+    const amount = web3.utils.toWei(params[2]);
+    const rateModeInput = params[3];
+    const user = params[4];
+
+    if (!isValidRateMode(rateModeInput)) return;
+    const rateMode = getRateModeNumber(rateModeInput);
+
+    if (privateKeyRequired) {
+      pk = params[5];
+      if (!pk) {
+        console.error('Missing private key');
+        return;
+      }
+      kit.addAccount(pk);
+    }
+
+    const repayDelegationHelperAddress = repayDelegationHelper.options.address;
+    try {
+      // check and approve spend of the tokens
+      const currentAllowance = await asset.methods
+        .allowance(user, repayDelegationHelperAddress)
+        .call();
+      if (currentAllowance < amount) {
```

Comment:
those value are either string or big numbers, in both cases lessThan operator wouldn't work as expected.